k8s/templates/job-template.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "studio.fullname" . }}-django-config
+  labels:
+    app: {{ template "studio.fullname" . }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation
+data:
+  DJANGO_SETTINGS_MODULE: {{ .Values.settings }}
+  DJANGO_LOG_FILE: /var/log/django.log
+  MPLBACKEND: PS
+  STUDIO_BETA_MODE: "yes"
+  RUN_MODE: k8s
+---
+apiVersion: v1
+kind: Secret
```

Comment:
I think it's great to put the secrets into the job-template.yaml for this PR. But I'm wondering if in the future, if we would like to add more secrets that do not apply to the job, will it be better to move the secrets (maybe the configMap as well) outside the job template?